src/ImpactEvaluator.sol

```diff
@@ -82,48 +79,41 @@ contract ImpactEvaluator is AccessControl {
             addresses.length == scores.length,
             "Addresses and scores length mismatch"
         );
-        require(roundIndex < currentRoundIndex(), "Round not finished");
+        require(roundIndex < currentRoundIndex, "Round not finished");
 
-        Round storage round = getOpenRound(roundIndex);
+        (uint openRoundsIndex, Round storage round) = getOpenRound(roundIndex);
```

Comment:
I am confused about the meaning of "openRoundsIndex" - how can multiple rounds share the same index? 

Is this referring to the index in the "open rounds" array? Maybe "openRoundsPosition" or "indexInOpenRounds" would be a better name?

Reply:
https://github.com/Meridian-IE/impact-evaluator/pull/50